### DISCUSSION TITLE
The Xfce themes. ALL of them.

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2466] [Show and tell]
Yeah. ALL the <a href=https://github.com/AzumaHazuki/lxqt-themes-xfce>old-school Xfce GTK2 themes</a>. Okay, _except_ Xfce-flat, 'cause it's ugly and doesn't look different enough from Xfce-dusk to justify its existence =P Each theme comes with its own matching palette, as usual.

This took a looooooong time, and there was a lot of detail work that had to be done. The themes, despite being very simple gtkrc files originally, produced some surprisingly complex-looking environments. It took hours of wracking my brain to figure out how to do the scrollbar groove cleanly in Xfce-winter, for example, and some of the arrows actually lose their corners when pressed. Odd.

![Xfce-4 0](https://github.com/lxqt/lxqt/assets/67122280/ced66d38-d11a-455f-b5f7-cf65ad39e1be)
![Xfce-4 2](https://github.com/lxqt/lxqt/assets/67122280/9078e6e5-873a-4820-9bfc-6c0f47325a80)
![Xfce-4 4](https://github.com/lxqt/lxqt/assets/67122280/754a6efe-5165-4a02-8114-1a1ad09b7836)
![Xfce-4 6](https://github.com/lxqt/lxqt/assets/67122280/1d0b261b-3724-4fa1-a976-fe4e81d89101)
![Xfce-b5](https://github.com/lxqt/lxqt/assets/67122280/80616441-207b-4b93-b68b-c0971e47455e)
![Xfce-basic](https://github.com/lxqt/lxqt/assets/67122280/1f67231b-1cd6-4469-b4ac-254749579d69)
![Xfce-cadmium](https://github.com/lxqt/lxqt/assets/67122280/c2742940-3512-4175-a90f-64373a2f1f6d)
![Xfce-curve](https://github.com/lxqt/lxqt/assets/67122280/0a6a42b3-9156-403b-af29-bdd8e61a11d0)
![Xfce-dawn](https://github.com/lxqt/lxqt/assets/67122280/7e14fa16-6528-4616-8bef-dfe0a4de7ec1)
![Xfce-dusk](https://github.com/lxqt/lxqt/assets/67122280/7bf80a97-e940-4e7f-bcbc-b7798bd2bb92)
![Xfce-kde2](https://github.com/lxqt/lxqt/assets/67122280/409349d6-76aa-44cb-96f3-a20c9cf027b3)
![Xfce-kolors](https://github.com/lxqt/lxqt/assets/67122280/ba563505-8eba-4174-a76a-ddf2b4b28b4f)
![Xfce-light](https://github.com/lxqt/lxqt/assets/67122280/8f139179-ac45-48f2-8bf0-c652a8138bdf)
![Xfce-orange](https://github.com/lxqt/lxqt/assets/67122280/f3034b31-74b3-4af9-be15-23c0b69eeb47)
![Xfce-redmondxp](https://github.com/lxqt/lxqt/assets/67122280/3ff3737f-ec0e-459e-afea-abc4f17ed651)
![Xfce-saltlake](https://github.com/lxqt/lxqt/assets/67122280/5e38bc22-1e6c-45a0-bfa4-1b14207ca5fa)
![Xfce-smooth](https://github.com/lxqt/lxqt/assets/67122280/02a6b4ef-b8d9-453b-a272-f79293eddad3)
![Xfce-stellar](https://github.com/lxqt/lxqt/assets/67122280/e219670c-fb29-4208-bfcd-7dbbb69aac59)
![Xfce-winter](https://github.com/lxqt/lxqt/assets/67122280/3192b261-bb2e-4bc4-8b18-813c5132e847)

**Comment by khumba** [comment]
Hello!  Where did this repo disappear to?  I hope there isn't a problem with distributing these, they are really well made.

**Reply by AzumaHazuki** [reply]
Aww, thanks :) The themes are being redone in spare time (and this is sparse -- I work in medicine in the United States, for extremely low pay, which as you can imagine takes a lot of time and mental energy!).

The themes as they were had no support for Fancy Menu, i.e., for LXQt >=2.0 they were broken, and there was one minor but annoying bug to do with the QSlider {} styling that was affecting the panel preferences. I just have to finish redoing the scrollbar theming for the set and they'll be done and re-uploaded, but it's going to take a bit...
